.env.example

```diff
@@ -0,0 +1,8 @@
+# Full path to the LEGO Universe client
+CLIENT_PATH=/Users/someuser/LEGO Universe
+# Can improve build time
+BUILD_THREADS=1
+# Updates NET_VERSION in CMakeVariables.txt
+BUILD_VERSION=171022
+# make sure this is a long random string
+ACCOUNT_MANAGER_SECRET=
```

Comment:
Is there any way this could be autogenerated at runtime by default (eg via `python -c 'import secrets; print(secrets.token_hex())'` as recommended by the Flask docs)? That way you don't need to rely on the end user knowing how to properly generate a cryptographically secure key.

Reply:
If it's not possible, this would have to be another setup step, e.g. run `openssl rand -base64 30` or [`Get-Random`](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/get-random?view=powershell-7) in Powershell. If we just give the user the command to execute this should be okay?

Reply:
Or we just add ad note to a page like https://randomkeygen.com/ or https://keygen.io/ or any other page.

Reply:
Yes, that's better. Good idea.

Reply:
Alternatively, if there are eventually a lot of steps to be done for setup, this could be done via a setup-script, which could also e.g. add the admin user.

Reply:
I added a comment in my PR to keygen.io. I like the user having control over any secrets.